feconf.py

```diff
@@ -31,7 +31,7 @@
 # This variable is for serving minified resources
 # when set to false. It reflects if we are running the server in the dev
 # environment.
-FORCE_DEV_MODE = True
+FORCE_PROD_MODE = False
```

Comment:
Please make sure to update the comments/docstrings for whatever you change.

Reply:
updated